src/main/kotlin/racingcar/domain/RacingCar.kt

```diff
@@ -1,13 +1,34 @@
 package racingcar.domain
 
+class RacingCars(
+    var list: List<RacingCar>
+) {
+
+    fun getWinners(): List<String> {
+        getMaxProgress().let { maxProgress ->
+            return list.filter { car -> car.progress == maxProgress }.map { it.driver }
```

Comment:
scope function 외부에서 return을 해주는게 더 좋아보입니다 
```suggestion
       return getMaxProgress().let { maxProgress ->
            list.filter { car -> car.progress == maxProgress }.map { it.driver }
        }
```

Reply:
@vsh123 
외부에서 처리하도록 변경하였습니다!
[65fa10f]